src/main/java/net/minecraftforge/common/ForgeModContainer.java

```diff
@@ -113,6 +113,7 @@
     public static boolean disableVersionCheck = false;
     public static boolean forgeLightPipelineEnabled = true;
     public static boolean replaceVanillaBucketModel = true;
+    public static boolean minimizeItemModelQuads = false;
```

Comment:
I don't think the config is necessary now that there are no artifacts.

Reply:
There's still the matter of the changes in 62d0f012361a05e9a6a5bf71f58fe4d5b1b1df59 no longer being applied.

Reply:
What is that commit fixing exactly, and why does this PR reintroduce the problem?